pkg/entity/data_subject/cmd.go

```diff
@@ -0,0 +1,65 @@
+package data_subject
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"strings"
+	"strmprivacy/strm/pkg/common"
+)
+
+func ListCmd() *cobra.Command {
+	longDoc := `Query the Data Subjects service for a list of data-subjects.
+
+Returns paginated data. If one page of data has following pages, its 'next_page_token'
+field must be added to the following call via the '--page-token' flag.
+`
+	outputFormatFlagAllowedValues := []string{common.OutputFormatPlain, common.OutputFormatPlain + "0",
+		common.OutputFormatJson, common.OutputFormatJsonRaw}
+	outputFormatFlagAllowedValuesText := strings.Join(outputFormatFlagAllowedValues, ", ")
+	command := &cobra.Command{
+		Use:               "data-subjects",
+		Short:             "List a page of data subjects",
+		Long:              longDoc,
+		DisableAutoGenTag: true,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			printer = configurePrinter(cmd)
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			list(cmd)
+		},
+	}
+	flags := command.Flags()
+	flags.Int32(pageSizeFlag, 0, "maximum number of items to be returned")
+	flags.String(pageTokenFlag, "", `page token to be entered for next page.
+Use the nextPageToken (if any) returned from the previous call`)
+	flags.StringP(common.OutputFormatFlag, common.OutputFormatFlagShort, common.OutputFormatPlain,
+		fmt.Sprintf("output format [%v]", outputFormatFlagAllowedValuesText))
+	return command
+}
+
+func DeleteCmd() *cobra.Command {
+	longDoc := `Deletes 1 or more data subjects from the Data Subjects Service.
+
+Returns the number of deleted key links that were associated with these data subjects.
+`
+
+	outputFormatFlagAllowedValues := []string{common.OutputFormatPlain,
+		common.OutputFormatJson, common.OutputFormatJsonRaw}
+	outputFormatFlagAllowedValuesText := strings.Join(outputFormatFlagAllowedValues, ", ")
+	command := &cobra.Command{
+		Use:               "data-subjects <data-subject-id>...",
```

Comment:
`<data-subject-id>` implies 1 value right, rather have something like `[id1,id2,..]`

Reply:
I see the `...` as multiple. Let's see what Unix does.
```
man ls

LS(1)                            User Commands                           LS(1)

NAME
       ls - list directory contents

SYNOPSIS
       ls [OPTION]... [FILE]...
```